doc/stages/writers.ept_addon.rst

```diff
@@ -0,0 +1,66 @@
+.. _writers.ept_addon:
+
+writers.ept_addon
+=================
+
+The **EPT addon writer** supports writing additional dimensions to `Entwine Point Tile`_ datasets.  Thus the EPT addon writer is unique among most other PDAL writers due to its limitation that it may only be used along with an `EPT reader`_ and modifies and existing dataset rather than creating a brand new one.
+
+The addon dimensions created by this writer may be stored independently from the corresponding EPT dataset, therefore write-access to the EPT resource itself is not required to create and use addon dimensions.
+
+.. embed::
+
+Example
+--------------------------------------------------------------------------------
+
+This example downloads the Autzen dataset (10M points) and runs the `SMRF filter`_, which populates the ``Classification`` dimension with ground values, and writes the resulting attribute to an EPT addon dataset on the local filesystem.
+
+.. code-block:: json
+
+   [{
+      "type": "readers.ept",
+      "filename": "http://na.entwine.io/autzen/ept.json",
+      "threads": 12
+   }, {
+      "type": "filters.assign",
+      "assignment": "Classification[:]=0"
+   }, {
+      "type": "filters.smrf"
+   }, {
+      "type": "writers.ept_addon",
+      "addons": { "~/entwine/addons/autzen/smrf": "Classification" },
```

Comment:
Also, to be somewhat consistent with what other stages do (e.g., filters.assign), I wonder if it would make more sense to allow repeated specification of `addons`. For example,

```
"addons": "~/entwine/addons/autzen/smrf=Classification",
"addons": "~/entwine/addons/autzen/normal_x=NormalX"
```

Although, we already differ between filters.ferry and filters.assign, as the filters.ferry way of doing this would be

```
"addons":"~/entwine/addons/autzen/smrf<=Classification,~/entwine/addons/autzen/normal_x=NormalX"
```

Reply:
If you the argument type is vector<string>, it will automatically accept comma-separated or repeated arguments.

Reply:
I'm strongly opposed to repeated arguments, which although technically valid JSON, have undefined parsing behavior and cannot be generated by JSON libraries in general.  I also tend to prefer the proposed native JSON mapping rather than a CSV string - I think it makes more sense from a JSON API standpoint even if asymmetrical from other filters whose interfaces were defined prior to PDAL JSON support.